locale/ar_IQ/emailTemplates.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE email_texts SYSTEM "../../../../../lib/pkp/dtd/emailTemplateData.dtd">
+
+<!--
+  * @file plugins/generic/sword/locale/ar_IQ/emailTemplates.xml
+  *
+  * Copyright (c) 2013-2016 Simon Fraser University Library
```

Comment:
All the copyright end dates and names need to be consistent.

Reply:
updated.